packages/core/botpress/src/config-manager/module.js

```diff
@@ -51,6 +51,18 @@ You can provide this value in "${filePath}"`
     return config
   }
 
+  saveAll(newConfig) {
+    const configPath = path.join(this.configLocation, this._getFileName())
+    const oldConfig = JSON.parse(fs.readFileSync(configPath, 'utf8'))
+
+    const config = Object.assign(oldConfig, newConfig)
+    fs.writeFile(configPath, JSON.stringify(config), 'utf8', err => {
```

Comment:
@dmytropanontko , you are using `readFileSync` and `writeFile`. I'd suggest to use sync methods here (given this operation isn't frequent) or [promisify](http://bluebirdjs.com/docs/api/promise.promisifyall.html) fs-methods and use them as promises.